src/components/Header/UserMenu.jsx

```diff
@@ -20,7 +20,12 @@ import { DeleteComfirmationModal } from '../DeleteComfirmationModal';
 
 export const UserMenu = () => {
   const { isOpen, onOpen, onClose } = useDisclosure();
-  const iconsThemeColor = useColorModeValue('black', 'white');
+  const userMenuBgThemeColor = useColorModeValue('orange.50', 'teal.800');
+  const userMenutextcolor = useColorModeValue('teal.900', 'orange.100');
+  const userMenuItemHoverBgColor = useColorModeValue('orang.100', 'teal.600');
```

Comment:
```suggestion
  const userMenuItemHoverBgColor = useColorModeValue('orange.100', 'teal.600');
```
I believe it's a typo